sessions/js-forms-2/simple-todolist/README.md

```diff
@@ -0,0 +1,27 @@
+# Simple To-do-List
+
+Everyone has something to do. Let's make sure that no one forgets anything.
+
+## Task
+
+### 1. Listen to the submit event
+
+- Write an event listener for the form submit.
+- Log the user data to the console (we will use the data later in this exercise).
+- After submit, reset the form and focus the input field with the name `headline`.
+
+### 2. Create list entries
+
+Look at the index.html file: there already is an `ul` for our submitted todos. We only need to write a function which creates list items and appends them to this list.
+
+Here are some hints to guide you:
+
+- The function needs one parameter (which will be the form data object).
+- The list item
+  - has a headline and a text element ("task").
+  - `textContent` of headline and task are set via the parameter (you can use dot notation: `data.headline` and `data.task`).
+- Don't forget to
+  - append the headline and text element to the list item as well as the list item to the todoList
+  - call the function when the submit event is fired and pass the data object - which you are currently logging to the console - as argument.
```

Comment:
I got a weird formatting in which the nested sentence looked like another task in the list, so I would change the structure of the sentence:
```suggestion
  - call the function when the submit event is fired and pass the data object as an argument.
```